src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/PackageSearchArgs.cs

```diff
@@ -40,7 +44,34 @@ public int VerifyInt(string number, int defaultValue)
                 return verifiedNumber;
             }
 
-            throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Strings.Error_invalid_number, number));
+            throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Strings.Error_invalidOptionValue, number, option));
+        }
+
+        private bool IsJsonFormat(string format)
+        {
+            if (!string.IsNullOrEmpty(format) && string.Equals(format, "json", StringComparison.CurrentCultureIgnoreCase))
+            {
+                return true;
+            }
+
+            return false;
```

Comment:
This should be equivalent and more concise. Be sure to test it.

```suggestion
            return !string.IsNullOrEmpty(format) && string.Equals(format, "json", StringComparison.CurrentCultureIgnoreCase)
```